SolidStateGroup/bullet-train-api
# feat: Roles management

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Added a "Roles" tab in Manage > Members
- Created a modal for creating and editing roles, along with editing their permissions based on their level (organisation, project, environment)
- Added "show_role_management" flag to wrap this feature.
## How did you test this code?



- Go to manage -> members.
- Click the "Roles" tab.
- Click the "Create Role" button.
- Create a new role.
- Select the role in the role list.
- Edit name and description. (If you want to exit without saving, a confirmation modal will appear.)
- Edit organisation, project, and environment permissions (If you want to exit or change the tab without saving, a confirmation modal will appear.)
- Click the trash button to delete the role.
